src/tools/ceph_objectstore_tool.cc

```diff
@@ -3058,7 +3062,10 @@ int main(int argc, char **argv)
     }
 
     bufferlist bl;
-    map_epoch = PG::peek_map_epoch(fs, pgid, &bl);
+    r = PG::peek_map_epoch(fs, pgid, &map_epoch, &bl);
+    if (r < 0)
+      cerr << "peek_map_epoch returns an error" << std::endl;
```

Comment:
We need to initialize map_epoch to 0.  We should disallow a pg import with a ``map_epoch == 0``, while allowing recovery of data using ``rados import``.  I would fail a rm-past-intervals in the error case.